app/src/org/commcare/android/view/EntityView.java

```diff
@@ -54,6 +56,7 @@
     public static final String FORM_IMAGE = "image";
     public static final String FORM_GRAPH = "graph";
     public static final String FORM_CALLLOUT = "callout";
+    private static final int CASE_LIST_IMAGE_MAX_DIMEN = 150;

```

Comment:
I chose this as a number that just seemed reasonable and wasn't causing crashes with a bunch of cases. I definitely don't think it should be any bigger because the case list images only every show up as very small anyway, but possible that it should be smaller? If anyone wants to weight in, feel free.